src/bindings/c/include/openvino/c/ov_compiled_model.h

```diff
@@ -88,7 +88,7 @@ ov_compiled_model_set_property(const ov_compiled_model_t* compiled_model, const
 OPENVINO_C_API(ov_status_e)
 ov_compiled_model_get_property(const ov_compiled_model_t* compiled_model,
                                const ov_property_key_e key,
```

Comment:
I think key can be `char*`

Reply:
If so, it will permit user pass any contents string as a property key, does it make sense?

Reply:
Additionally, it will lead to any property key typo cannot be detected during compile stage.

Reply:
But we can introduce `static const char*` members which can be used for setting properties (the same as in old C++)

Reply:
It seems your idea doesn't work. 
For example, if user want to set "NUM_STREAMS", but user typed it as "NUM_STREAM" (missing a "S"), it still pass compilation, but will throw error during running.

Reply:
Yes, it is why I propose to introduce some constants like:
```
static const char* ov_property_num_streams = "NUM_STREAMS";
```
And user can use these properties on the application side:
```
ov_compiled_model_get_property(model, ov_property_num_streams, ...);
```

Reply:
oh, I got your point, but it have any essential difference with enum type?

Reply:
Yes, at least two:

- It is more flexible
- We can provide separate header with device specific properties for each device and don't mix device specific and common options.

Reply:
Enumerate type can also implement it as well :)

Reply:
I agree that it is hard to handle 3rd plugin property by enum.
Ok, let adopt const string to expose property key.

Reply:
done